src/js/getIllustrations.js

```diff
@@ -0,0 +1,35 @@
+export default function getIllustrations(bookSlug) {
+
+  switch (bookSlug) {
+    case 'mat':
+      return {
+        picker: require('../assets/Book_Illustrations/Matthew_Picker.jpg'),
+        sketch: require('../assets/Book_Illustrations/Matthew_Sketch_Medium.jpg'),
+      };
+
+    case 'mrk':
+      return {
+        picker: require('../assets/Book_Illustrations/Mark_Picker.jpg'),
+        sketch: require('../assets/Book_Illustrations/Mark_Sketch.jpg'),
+      };
+
+    case 'luk':
+      return {
+        picker: require('../assets/Book_Illustrations/Luke_Picker.jpg'),
+        sketch: require('../assets/Book_Illustrations/Luke_Sketch.jpg'),
+      };
+
+    case 'jhn':
+      return {
+        picker: require('../assets/Book_Illustrations/John_Picker.png'),
+        sketch: require('../assets/Book_Illustrations/John_Sketch.jpg'),
+      };
+
+    default:
+      return {
+        picker: require('../assets/Book_Illustrations/default.jpg'),
+        sketch: require('../assets/Book_Illustrations/default.jpg'),
+
+      };
+  }
+}
```

Comment:
what do you think about renaming the files with the slug at the beginning and reduce the code to around 16 lines??  or  use getIllustration(bookName)... I do not know if that is an option

export default function getIllustrations(bookSlug) {

 if(bookSlug) {
   return {
       picker: require(`../assets/Book_Illustrations/${bookSlug}_Picker.jpg`),
       sketch: require(`../assets/Book_Illustrations/${bookSlug}_Sketch_Medium.jpg`),
     };
 }
 else {
   return {
       picker: require('../assets/Book_Illustrations/default.jpg'),
        sketch: require('../assets/Book_Illustrations/default.jpg'),

      };
 }
}

Reply:
Oh I get you. Yeah that's a good idea. Don't think we can use book name because I think that will change for different languages if i'm not mistaken, but the book slugs should always stay the same. I'll shorten the function and push it back up.